### PR TITLE
Brett Madrid

### PR DESCRIPTION
**Pull request by brettmadrid**
MVP complete for the first day of this project.

### CODE REVIEW

File: users/users-model.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+module.exports = {
+  add,
+  find,
+  findBy,
+  findById,
+};
+
+function find() {
+  return db('users').select('id', 'username', 'password');
```
Comment (johnoro) [inline users/users-model.js:15]: Is the `select` needed when all columns of the table are being selected? You could also optionally just use a [wildcard](https://en.wikipedia.org/wiki/Wildcard_character) to select all columns:
```javascript
  return db('users').select('*');
```
but I say, why even have the extra step/method?
```javascript
  return db('users');
```

File: users/users-model.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+async function add(user) {
+  const [id] = await db('users').insert(user);
+
+  return findById(id);
+}
+
+function findById(id) {
+  return db('users')
+    .where({ id })
```
Comment (johnoro) [inline users/users-model.js:30]: Why not use your `findBy` helper here? For example:
```javascript
  return findBy({ id }).first();
```

File: server.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      res.status(500).json(error);
+    });
+});
+
+server.get("/api/users", restricted, (req, res) => {
+  Users.find()
+    .then(users => {
+      res.json(users);
+    })
+    .catch(err => res.send(err));
```
Comment (johnoro) [inline server.js:85]: You should specify a status here. The default when data is sent with no `status` method is 200, which signifies that all's `OK`, not that an error occurred!

**Review by johnoro — APPROVED** [review]
Lookin' good!

File: users/users-model.js (hunk 1 above)
Comment (brettmadrid) [reply users/users-model.js:15]: I like it.  I also didn't know I could use a wildcard.  Thanks

File: users/users-model.js (hunk 2 above)
Comment (brettmadrid) [reply users/users-model.js:30]: That actually makes a lot of sense.  Making that change now

File: server.js (hunk 3 above)
Comment (brettmadrid) [reply server.js:85]: I will add that now.

**Pull request merged**